latest/swarm/google/cloud-deployment.yaml

```diff
@@ -99,5 +99,25 @@ resources:
               workerImage: ranlu/air-tasks:airflow2
               preemptible: True
               sizeLimit: 4
+            - type: synaptor-cpu
+              zone: us-east1-c
+              subnetwork: us-east1
+              machineType: e2-standard-32
+              preemptible: True
+              sizeLimit: 10
+            - type: synaptor-gpu
+              zone: us-east1-c
+              subnetwork: us-east1
+              machineType: n1-highmem-8
+              gpuWorkerAcceleratorType: nvidia-tesla-t4
+              preemptible: True
+              sizeLimit: 10
+            - type: synaptor-seggraph
+              zone: us-east1-c
+              subnetwork: us-east1
+              machineType: n1-highmem-32
```

Comment:
Any particular reason to choose n1 instances? not newer types like n2 or e2?

Reply:
I just haven't looked into changing it. It's probably worth changing this before any big runs, so I'll try to make another choice in a few days before I merge this.

Reply:
That is fine, segmentation pipeline has a similar situation, the instance type of `composite` cluster should be adjusted for big run.

Reply:
Understood. I've swapped in more economic instance types now anyway.